packages/grafana-data/src/types/index.ts

```diff
@@ -52,14 +52,4 @@ export * from './alerts';
 export * from './slider';
 export * from './accesscontrol';
 export * from './icon';
-export {
-  type PluginExtension,
-  type PluginExtensionLink,
-  isPluginExtensionLink,
-  assertPluginExtensionLink,
-  type PluginExtensionCommand,
-  isPluginExtensionCommand,
-  assertPluginExtensionCommand,
-  PluginExtensionTypes,
-  PluginExtensionPlacements,
-} from './pluginExtensions';
+export * from './pluginExtensions';
```

Comment:
Please see the [other comment](https://github.com/grafana/grafana/pull/65327/files#r1153239821) about named exports.

Reply:
Good point 👍